src/Angor/Avalonia/Angor.Projects/Application/Dtos/ProjectDto.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Angor.Projects.Domain;
+
+namespace Angor.Projects.Application.Dtos;
+
+public record ProjectDto
+{
+    public ProjectId Id { get; set; }
+    public string Name { get; set; }
+    public string ShortDescription { get; set; }
+    public Uri? Picture { get; set; }
+    public Uri? Icon { get; set; }
+    public long TargetAmount { get; set; }
+    public DateOnly StartingDate { get; set; }
+    public TimeSpan PenaltyDuration { get; set; }
+    public string NpubKey { get; set; }
```

Comment:
We should keep the hex and parse the key from it, the key itself is not a string if I remember correctly.

Reply:
OK. I've renamed the properties to match the existing.